src/modules/launcher/Wox.Plugin/Common/VirtualDesktop/VirtualDesktopHelper.cs

```diff
@@ -101,22 +106,23 @@ public void UpdateDesktopList()
             }
             else
             {
-                Log.Error("Failed to read the list of existing desktops form registry.", typeof(VirtualDesktopHelper));
+                Log.Debug("VirtualDesktopHelper.UpdateDesktopList() failed to read the list of existing desktops form registry.", typeof(VirtualDesktopHelper));
             }
 
             // Guid for current desktop
-            int userSessionId = Process.GetCurrentProcess().SessionId;
-            RegistryKey currentDeskSubKey = Registry.CurrentUser.OpenSubKey($"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\SessionInfo\\{userSessionId}\\VirtualDesktops", false);
-            var currentDeskValue = currentDeskSubKey.GetValue("CurrentVirtualDesktop");
+            var currentDeskSessionValue = Registry.CurrentUser.OpenSubKey(registrySessionVirtualDesktops, false).GetValue("CurrentVirtualDesktop", null); // Windows 10
+            var currentDeskExplorerValue = Registry.CurrentUser.OpenSubKey(registryExplorerVirtualDesktops, false).GetValue("CurrentVirtualDesktop", null); // Windows 11
+            var currentDeskValue = currentDeskSessionValue ?? currentDeskExplorerValue;
```

Comment:
Should be validated based on OS version to prevent problems after upgrade/downgrade.

Reply:
done with commit 0fba547